apps/sms/views/conversation/js/compose.js

```diff
@@ -679,7 +679,7 @@ var Compose = (function() {
 
       var currentAttachmentDOM = event.target;
       var currentAttachment = attachments.get(event.target);
-      var name = currentAttachment.name;
+      var name = currentAttachment.name || '';
```

Comment:
it's likely not necessary with the code we have in attachment.js :) I know this was my first proposal but this was before I properly identified the issue.